gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
It's nice to also add [SHA checksum verification](https://docs.gradle.org/current/userguide/gradle_wrapper.html#configuring_checksum_verification)

Reply:
Will do

Reply:
Fixed